tools/db_bench_tool.cc

```diff
@@ -5418,10 +5418,12 @@ class Benchmark {
       if (key_dist_a == 0.0 && key_dist_b == 0.0) {
```

Comment:
It would be safe to change it to if (key_dist_a == 0.0 || key_dist_b == 0.0), to avoid x/0.

Reply:
Yeah, you are right! I'll add another commit.